Node/src/TeamsChatConnector.ts

```diff
@@ -110,6 +112,17 @@ export class TeamsChatConnector extends builder.ChatConnector {
   }
 
   /**
+  * This method is to resolve regex or LUIS Bot being affected by at mentions
+  * Set true would have incoming messages stripped off source Bot at mentions
+  * The default would be false
```

Comment:
Can you explain why this is minimum change and easier to maintain? Developers are currently using the middleware approach, so if anything it seems that would be the smaller change from their POV. (I like that this is potentially more discoverable though.)